tests/lib/Item/ValueConverter/ProductValueConverterTest.php

```diff
@@ -74,6 +74,12 @@ public function testGetIsVisible(): void
                 new ProductStub(42, 'Product name'),
             ),
         );
+
+        self::assertFalse(
```

Comment:
This should go to a separate test, e.g. `testGetIsVisibleReturnsFalse()`.

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/13/commits/c1ab34cf3634525aea39589988e5c69e334e2744